solr/core/src/java/org/apache/solr/core/FileSystemConfigSetService.java

```diff
@@ -259,6 +260,14 @@ public FileVisitResult postVisitDirectory(Path dir, IOException ioException) {
     return filePaths;
   }
 
+  private String normalizePathToForwardSlash(String path) {
+    return path.replace(File.separatorChar, '/');
```

Comment:
A short-circuit to do nothing if these are the same; would be nice but don't have to.